packages/experiments/src/components/SelectedItemsList/Items/EditableItem.tsx

```diff
@@ -19,19 +20,15 @@ export type EditableItemProps<T> = {
 // `extends any` to trick the parser into parsing as a type decl instead of a jsx tag
 export const EditableItem = <T extends any>(editableItemProps: EditableItemProps<T>): Item<T> => {
   return React.memo((selectedItemProps: ISelectedItemProps<T>) => {
-    const [isEditing, setIsEditing] = React.useState(false);
-    const setEditingTrue = React.useCallback(() => {
-      setIsEditing(true);
-    }, [setIsEditing]);
-    const setEditingFalse = React.useCallback(() => {
-      setIsEditing(false);
-    }, [setIsEditing]);
+    const { onItemChange, index } = selectedItemProps;
+    const [isEditing, { setTrue: setEditingTrue, setFalse: setEditingFalse }] = useBoolean(false);
```

Comment:
The change to `useBoolean` will have the exact same behavior as before, with less verbosity (it was made for cases like this)